document_page/README.rst

```diff
@@ -0,0 +1,100 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Document Page
+==============
+
+This module allows you to write web pages for internal documentation.
+
+Installation
+============
+
+This module depends on module knowledge. So make sure to have available it in your addons list
+
+Configuration
+=============
+
+No configuration required
+
+Usage
+=====
+
+To use this module, you need to:
+* Install the module's dependencies
+* Install this module module
+* Got to knowledge menu
+* use submenu Categories to Create the document's category you need
+* use submenu Pages to  Create pages for that category
+* use tu submenu Pages history for pages revision history
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "8.0" for example
+
```

Comment:
The target link should be https://runbot.odoo-community.org/runbot/118/9.0

You can also remove the comments.